docs/ref/settings.txt

```diff
@@ -1260,6 +1260,31 @@ Default: ``'``:class:`django.db.models.AutoField`\ ``'``
 Default primary key field type to use for models that don't have a field with
 :attr:`primary_key=True <django.db.models.Field.primary_key>`.
 
+.. admonition:: Migrating auto-created through tables.
+
+    The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
+    auto-created through tables for many-to-many relationships.
+
+    The primary key of auto-created through tables cannot currently be updated
+    by the migrations framework, however.
+
+    This means that if you switch the value of ``DEFAULT_AUTO_FIELD`` and then
+    generate migrations, the primary keys of the related models will be
+    updated, as will the foreign keys from the through table, but the primary
```

Comment:
I think we can chop `will` :think`
```suggestion
    updated, as the foreign keys from the through table, but the primary
```

Reply:
I think it is necessary. `as will` here ≈`same with` (but `same with` wouldn't work).

Reply:
I can think about a rephrase if you think it doesn't read.

If not, if likely my Britishness, and tendency to archaic forms, coming through — Happy to adjust!

Reply:
> If not, it's likely my Britishness, and tendency to archaic forms, coming through — Happy to adjust!

It reads correctly to me as you've written it. I wouldn't have thought it archaic.